scripts/integration-test.sh

```diff
@@ -0,0 +1,56 @@
+CURRENT_DIR=$(pwd)
+SANDBOX_PATH=$CURRENT_DIR/testnet/testnet-local/sandbox
+KEY_GENERATOR_PATH=$CURRENT_DIR/testnet/elrond-go/cmd/keygenerator
+
+setup(){
+  echo "starting integration tests for shard $1"
+
+  cd testnet/elrond-go/cmd/keygenerator
+  go build
+  ./keygenerator
+  cd ../../../../../devel/standard/
```

Comment:
suggestion to use `pushd` and `popd` here instead of so many `../..`

Reply:
Changed as suggested